vllm/model_executor/layers/mamba/ops/ssd_chunk_state.py

```diff
@@ -36,41 +36,35 @@ def _chunk_cumsum_fwd_kernel(
     dt_out_ptr,
     dA_cumsum_ptr,
     # Matrix dimension
-    batch,
     seqlen,
-    nheads,
-    chunk_size,
-    dt_min,
-    dt_max,
+    nheads: tl.constexpr,
+    chunk_size: tl.constexpr,
+    dt_min: tl.constexpr,
+    dt_max: tl.constexpr,
```

Comment:
What actually are these arguments used for? At first I thought these shouldn't be constexpr because they would me the min/max range of dt tensor. But it looks like they are hard-coded here: https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/layers/mamba/mamba_mixer2.py#L713

Should we just remove them?

Reply:
@fabianlim can maybe comment on this